.github/workflows/book.yml

```diff
@@ -1,10 +1,7 @@
 name: deploy-book
 
-# Run on pushes
-on:
-  push:
-    branches:
-      - '*'
+# Run on pushes & pull requests
+on: [push, pull_request]
```

Comment:
@kysolvik made this change in another PR so i just added it so we could build the docs here and see that all is good with sphinx!